scripts/customize_build_env.sh

```diff
@@ -180,8 +187,47 @@ set_center_specific_externals()
 
     if [[ ${center} = "llnl_lc" ]]; then
         case ${spack_arch_target} in
-            "broadwell" | "haswell" | "zen" | "power9le" | "power8le")
+            "broadwell" | "haswell" | "power9le" | "power8le")
+cat <<EOF  >> ${yaml}
+    rdma-core:
+      buildable: False
+      version:
+      - 20
+      externals:
+      - spec: rdma-core@20 arch=${spack_arch}
+        prefix: /usr
+EOF
+                ;;
+            "zen2")
```

Comment:
`spack arch -t` returns `zen` for me on corona.  We should probably have this as `"zen" | "zen2"`

Reply:
Yes, it depends on if you are on the head node or compute node.